dts/bindings/sensor/avago,apds9306.yaml

```diff
@@ -42,13 +42,13 @@ properties:
     type: int
     default: 100
     enum:
-      - 2000
-      - 1000
-      - 500
-      - 200
-      - 100
-      - 50
       - 25
+      - 50
+      - 100
+      - 200
+      - 500
+      - 1000
+      - 2000
```

Comment:
The property sets not a frequency value but a measurement period in milliseconds.
I think the property may be renamed to e.g. `measurement-period` (like [vishay,vcnl36825t](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/sensor/vishay%2Cvcnl36825t.yaml#L26))

Reply:
Renaming the field causes issues with the Twister tests
https://github.com/zephyrproject-rtos/zephyr/actions/runs/15822735532/job/44595853980?pr=91105